docs/source/inference/moments_inference.rst

```diff
@@ -1,20 +1,10 @@
-=========
-Inference
-=========
-
-Here we will have standard error functions for the following types of models:
-
-- maximum likelihood (ml) generalized method of moments (gmm) method of simulated
-  moments
-- (msm) indirect inference (ii)
-
-Generalized Method of Moments (GMM)
------------------------------------
+Moments Based Estimation
+========================
 
 .. currentmodule:: estimagic.inference.moment_covs
 
-In this section we document how calculate standard errors of a (GMM) estimator. We use
-the notation and formulations provided in section 13.25 of
+In this section we document how calculate standard errors of a GMM or MSM estimator.
+We use the notation and formulations provided in section 13.25 of
 
 Bruce E. Hansen - Econometrics (https://www.ssc.wisc.edu/~bhansen/econometrics)
```

Comment:
different way to write the reference from how it is done in numerical differentiation. We should choose one for consistency but I don't have a preference.

Reply:
Now citations are done properly using sphinxcontrib.bibtex and a `refs.bib` file that collects all references.